modules/cnf-numa-resource-scheduling-strategies.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: CONCEPT
+[id="cnf-numa-resource-scheduling-strategies_{context}"]
+= NUMA Resource scheduling strategies 
```

Comment:
Seems like it should be lowercase "resource" when it's a generic reference / not specifically referring to the "NUMA Resources Operator":
```suggestion
= NUMA resource scheduling strategies 
```